uint32 	msg_counter
bool 	isObject
float32 yaw_rate
